examples/todo-list/flutter-todo-list/lib/screens/signup.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:todosupabase/constant.dart';
+
+class SignUpPage extends StatefulWidget {
+  const SignUpPage({Key? key}) : super(key: key);
+
+  @override
+  State<SignUpPage> createState() => _SignUpPageState();
+}
+
+class _SignUpPageState extends State<SignUpPage> {
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+  final TextEditingController _nameController = TextEditingController();
+
+  Future<bool> createAccount({
+    required String email,
+    required String password,
+  }) async {
+    final res = await client.auth.signUp(email, password);
+    final error = res.error;
+    if (error == null) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  Future<dynamic> addUser({
+    required String name,
+    required String email,
+  }) async {
+    final res = await client.from('Users').insert({
+      'name': name,
+      'email': email,
+    }).execute();
+  }
```

Comment:
I believe with the official `Todo List` quick start, we only have a table named `todos`. Could we modify this sample app to use the officla `Todo List` quick start as well?

Reply:
Ok so if i replicate the db in the nextjs example where providers is used for authentication but I'm using signup with emailID so i guess we need a separate table right ?